src/main/java/io/reactivex/schedulers/Schedulers.java

```diff
@@ -243,9 +243,9 @@ public static Scheduler newThread() {
      * <p>
      * Uses:
```

Comment:
nit: I don't really understand use of "Uses" here… so if you have some suggestions, I could update this as well